modules/ocf_apphost/files/vhost-app.jinja

```diff
@@ -0,0 +1,38 @@
+# {{vhost.comment}}
+server {
+    listen {{vhost.port}};
+    listen [::]:{{vhost.port}};
+    server_name "{{vhost.fqdn}}";
+
+    # Proxy /.well-known/ to www.o.b.e/.well-known/ so death can request
+    # Let's Encrypt certificates for app vhosts
```

Comment:
It does make sense for the apphost to request its own certs, I might as well fix that.

I'm not really sure what scenario you have in mind where a server besides death or werewolves would want to request LE certs however.